### PR TITLE
add Markdown comonenet, change rpc docs to static

### LINKED ISSUE #72: RPC - DOCS - make static

### PR DESCRIPTION
**Pull request by Adam-Goldman7**
resolves #72

### CODE REVIEW

File: client/components/RpcDocs/index.js [hunk 1]
```diff
@@ -58,51 +58,29 @@ export default class RpcDocs extends Component {
...
-              <p key={`${m.name} 1`}>{m.desc}</p>,
-              <p key={`${m.name} 2`}><strong>Params - </strong>{m.params.length ? m.params : 'none'}</p>,
-              <p key={`${m.name} 3`}><strong>Returns - </strong>{m.returns}</p>,
-              <hr />
-            ]}
-          />
+            disabled
+            rel={m.name}
+          >
+            <h3 style={{textTransform: 'none'}}>{m.name}</h3>
```
Comment (tomusdrw) [inline client/components/RpcDocs/index.js]: inline styles :(

File: client/components/ScrollTopButton/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        onClick={() => scrollTo(document.body, 0, 300)}>
+        <ArrowUpwardIcon />
+      </IconButton>
+    );
+  }
+
+  handleScroll (event) {
+    let {scrollTop} = event.srcElement.body;
+
+    if (!this.state.showScrollButton && scrollTop > 600) {
```
Comment (tomusdrw) [inline client/components/ScrollTopButton/index.js]: why 600?

File: client/components/ScrollTopButton/util.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    let position = easeInOut(elapsedTime, start, change, duration);
+    element.scrollTop = position;
+    if (elapsedTime < duration) {
+      setTimeout(() => {
+        animateScroll(elapsedTime);
+      }, increment);
+    }
+  };
+
+  animateScroll(0);
```
Comment (tomusdrw) [inline client/components/ScrollTopButton/util.js]: I think we should stop animation if use scrolls the page.
(Imagine clicking "Back to top" and then finding what was interesting while it scrolls)

File: client/components/RpcDocs/index.js (hunk 1 above)
Comment (Adam-Goldman7) [inline client/components/RpcDocs/index.js]: yeah I didn't c much point putting it into a class

tho it is more future proof I guess ...

Will change

File: client/components/ScrollTopButton/index.js (hunk 2 above)
Comment (Adam-Goldman7) [inline client/components/ScrollTopButton/index.js]: just seemed reasonable

u have another number in mind?

Comment (tomusdrw) [inline client/components/ScrollTopButton/index.js]: Something relative to viewport height?

Comment (Adam-Goldman7) [inline client/components/ScrollTopButton/index.js]: not sure what u mean
example?

Comment (tomusdrw) [inline client/components/ScrollTopButton/index.js]: `if (... && scrollTop > this.viewportHeight())`

`viewportHeight() { return window.innerHeight; }`

**Pull request merged**